helloWorld.metta

```diff
@@ -0,0 +1 @@
+!("Hello World"!);
```

Comment:
@Simret this line of code insert data into atomspace. we should add println!("Hello Worl") to see terminal output